Source/Core/VideoCommon/VertexManagerBase.h

```diff
@@ -172,7 +174,9 @@ class VertexManagerBase
   u32 GetRemainingIndices(OpcodeDecoder::Primitive primitive) const;
 
   void CalculateZSlope(NativeVertexFormat* format);
-  void LoadTextures();
+
+  BitSet32 UsedTextures();
```

Comment:
```suggestion
  BitSet32 UsedTextures() const;
```

I believe this can be const